packages/tracer/src/Tracer.ts

```diff
@@ -321,7 +321,7 @@ class Tracer extends Utility implements TracerInterface {
    * @deprecated Use {@link captureAWSv3Client} instead.
    * @param service - AWS SDK v2 client
    */
-  public captureAWSClient<T>(service: T): T {
+  /* v8 ignore start */ public captureAWSClient<T>(service: T): T {
```

Comment:
What is the reason for v8 ignore directive? Deprecation?

Reply:
Yes, correct. The method is marked as deprecated (L321) and so I just removed the test that was using it.